spec/deprecation-cop-view-spec.coffee

```diff
@@ -37,7 +37,7 @@ describe "DeprecationCopView", ->
 
     pack = atom.packages.loadPackage(fakePackageDir)
     spyOn(atom.packages, 'getLoadedPackages').andReturn([pack])
-    deprecationCopView.find("button.refresh-selectors").click()
+    deprecationCopView.updateSelectors()
```

Comment:
Needed to manually call `updateSelectors` here because the package is just being loaded here, not activated. And only activations are being used as signals to refresh deprecated selectors: 

https://github.com/atom/deprecation-cop/pull/44/files#diff-6c4904ca8d901aba3f94a47ab0f091eeR30